src/server/Master.ts

```diff
@@ -146,6 +146,8 @@ app.get(
   "/api/env",
   gatekeeper.httpHandler(LimiterType.Get, async (req, res) => {
     const envConfig = {
+      /* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing --
+      Allow for empty string ('') value to be substituted by default "prod". */
       game_env: process.env.GAME_ENV || "prod",
```

Comment:
I think blank `GAME_ENV` should be treated as an error, so we should use `!` here.
```ts
const game_env = process.env.GAME_ENV;
if (!game_env) return res.sendStatus(500);
```
```suggestion
      game_env,
```

Reply:
Modified in 1707e90:
```ts
app.get(
  "/api/env",
  gatekeeper.httpHandler(LimiterType.Get, async (req, res) => {
    const envConfig = {
      game_env: process.env.GAME_ENV,
    };
    if (!envConfig.game_env) return res.sendStatus(500);
    res.json(envConfig);
  }),
);
```